recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
Those lines are already present below (26 - 27).

I think you need to add `omniorb    # [build_platform != target_platform]` so you have an `omniiidl`you can run. This is what we do in [cpptango](https://github.com/conda-forge/cpptango-feedstock/blob/681a5a578ceb1de9b92f5185c01f1d2780747623/recipe/meta.yaml#L30)